indi-maxdomeii/maxdomeii.cpp

```diff
@@ -701,21 +710,25 @@ bool MaxDomeII::ISNewNumber(const char *dev, const char *name, double values[],
             if (error == IPS_OK)
             {
                 nShutterOperationPosition             = nVal;
-                ShutterOperationAzimuthNP.s           = IPS_OK;
-                ShutterOperationAzimuthNP.np[0].value = nVal;
-                IDSetNumber(&ShutterOperationAzimuthNP, "New shutter operation azimuth set");
+                ShutterOperationAzimuthNP.setState(IPS_OK);
+                // FixMe
```

Comment:
Another pending Fix.

Reply:
Same.
No such line of code exists. It is updated in the latest code.